README.md

```diff
@@ -1,5 +1,151 @@
+- As it turned out, a bit of my game spec seems to be a bit of a feature test anyway but my next step was to build the proper feature test.
+
+
+### Still to do
+
+- Make the interface clearer that a strike or spare has been scored
+- Update functionality for frame 10 with the additional ball - if a 10 is scored first then don't count the extra point. Also update index.html to allow 3 balls - only make the 3rd appear if relevant?
+- Make the user interface pretty including using flash to notify a problem
+- Do all the normal refactoring for single responsibility!!
+- Look to see if I could use better syntax anywhere - map or reduce, foreach instead of a for loop for arrays etc
+- Add error handling to stop the user adding more than 10 points per frame
+
+### User Stories
+
+I extracted the following user stories:
+
+```
+As a player
+So that I can record my turn at bowling
+I want to be able to enter the number I scored on a roll of the ball"
+
+As a player
+So that I can see what I scored for the frame
+I want to be able to enter up to two rolls for the frame"
+
+As a player
+So that I can correctly score my turns
+I want to be limited to only one entry if I score a strike"
+
+As a player
+So that I don't incorrectly enter information
+I want to be restricted to entering a maximum of 10 points for one frame"
+
+As a player
+So that I can see how I am doing
+I want to be able to see my current score at all times"
+
+As a player
+So that I can see how I am doing
+I want to be able to see the score for each frame"
+
+As a player
+So that I can see the correct score
+I want bonus points to be added for a strike"
+
+As a player
+So that I can see the correct score
+I want bonus points to be added for a spare"
+
+As a player
+So that I can end the game on a high
+I want to be able to enter up to 3 rolls on the final frame"
+
+As a player
+So that I can see the correct score
+I want the points to be added correctly for a final frame"
+
+As a player
+So that I can see how many pins I could knock down
+I want to be able to see how many pins are still remaining in the current frame"
+
+As a player
+So that I don't feel especially bad about my bowling
+I want to see an amusing animation when I score zero for the whole game"
+
+As a player
+So that I feel really good about winning
+I want to see a celebratory animation when I score strikes on every ball for the game"
+```
+
+### Objects
+
+Once I understood the user stories, I modelled the objects and their methods, using a sequence diagram (which I abandoned because it didn't seem to tell me much) and then a class diagram. **INSERT LINK**
+
+- Game object - to control the overall scorecard
+- Frame object - to control the score per frame
+- Bonus object - to keep track of required bonus points because of a strike or a spare - **NOW REMOVED AS IT WAS TOO COMPLICATED**
+
+### Data structures
+
+- The game holds:
```

Comment:
good to list responsibilities for each object. could also be combined with above if preferred.